backend/Modules/Users/HitMeApp.Users/UsersModuleExtensions.cs

```diff
@@ -24,6 +26,8 @@ public static IApplicationBuilder UseUsersModule(this IApplicationBuilder app)
             containerBuilder.RegisterInstance(logger).As<ILogger>().SingleInstance();
             UserModuleCompositionRoot.SetContainer(containerBuilder.Build());
 
+            app.RegisterExceptionMapper<UsersModuleExceptionMapper>(@"HitMeApp.Users.*");
```

Comment:
These regexes might fail. It would match namespace e.g. `HitMeAppQUsersfoo`. I believe you meant it to be `HitMeApp\.Users\..+` or  `HitMeApp\.Users.*`, depending whether you want to match **only** deeper namespaces or not.

Reply:
Yeah. I just kind of assumed that we follow the convention all along hence the naive regex. I'll enforce a more specific one then!

Reply:
This will look weird if we use `\.` everywhere indeed, maybe we could come up with some kind of `NamespaceRegexBuilder` where we'd pass `HitMeApp`, `Users` etc. and it would be concatenated with `\.` behind the curtain?